command/expand_test.go

```diff
@@ -328,6 +329,70 @@ func TestExpandSource_Do_Not_Follow_Symlinks(t *testing.T) {
 	assert.Equal(t, []string{workdirJoin}, expected)
 }
 
+func TestRawSourceWithoutSymLinks(t *testing.T) {
```

Comment:
Do we still need these tests?

Reply:
We use `rawSource` method in `expandSource` but it was due to eliminate code duplication. With the change in url package there is no need to use `rawSource` method, therefore I can remove it and change the `expandSource` method to the normal version.